ftw/subsite/configure.zcml

```diff
@@ -32,10 +34,12 @@
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
 
- <subscriber
-      for=".interfaces.ISubsite
-           Products.Archetypes.interfaces.IObjectInitializedEvent"
-      handler=".handlers.block_parent_portlets" />
+  <include package="ftw.upgrade" file="meta.zcml" />
```

Comment:
A trick I learned from @jone:

Import the upgrades here by including the package with `<include package=".upgrades" />`, then configure the upgrades directory in `ftw/subsite/upgrades/configure.zcml` *after* the existing upgrade steps:

``` 
<upgrade-step:directory
    profile="ftw.subsite:default"
    directory="."
/>
```

This way the old upgrade steps are displayed too.

---

Before:

![vorher](https://cloud.githubusercontent.com/assets/28220/8348800/7936dfe8-1b17-11e5-8b52-28db5f7c6691.png)

---

After:

![nachher](https://cloud.githubusercontent.com/assets/28220/8348793/64025df0-1b17-11e5-8381-968882c04273.png)

---